README.md

```diff
@@ -202,6 +202,38 @@ the tool by hand to make sure everything looks correct and the benchmark results
 9. Add at least one example of using the new format to the `EXAMPLES` string in `Main`. Copy this
 example into the `Examples` section of this README.
 
+### Adding an option for Ion Data Generator
+
+Adding an option to specify generated scalar type Ion Data involves the following steps.
+1. In `Main`, add the option to the `USAGE` and `OPTIONS` strings, mimicking the existing format.
+2. Add invalid options hashset to `GeneratorOptionsValidator` , mimicking the existing format.
+3. Add parsing options logic to `GeneratorOptions` and consider which type of Ion data the parsed parameters should be passed to, 
+   mimicking the existing logic of passing options parameters.
+4. In `WriteRandomIonValues`, add logic to the methods of constructing Ion Data to process the added option.
+
+### Adding more supported data type to Ion Data Generator
+
+Adding abilities to Ion Data Generator to generate more types of Ion Data which conform with the Ion Schema.
+1. In `ReadGeneralConstaints`, add more data type cases under switch logic, mimicking the existing supported data type.
+2. Add methods in `WriteRandomIonValues`, mimicking the existing methods which writes random ion data to the generated file. 
+   If the data type is already supported by the existing methods, find the relative method to see if more logic processing the constraints needed to be added.
```

Comment:
relative -> related

Reply:
> relative -> related

Changes will be updated in the next commit.